docs/source/api_other.rst

```diff
@@ -131,15 +129,8 @@ GUI
 .. autoclass:: intake.interface.catalog.select.CatSelector
    :members:
 
-.. autoclass:: intake.interface.source.defined_plots.Plots
```

Comment:
These both still exist. Perhaps they need panel in the environment to import successfully?